src/Utopia/Messaging/Adapters/SMS/ClickSend.php

```diff
@@ -41,6 +41,15 @@ protected function process(SMS $message): string
             fn ($to) => \ltrim($to, '+'),
             $message->getTo()
         );
+        $body = array();
+        foreach ($to as $t) {
+            array_push($body, [
+                'body' => $message->getContent(),
+                'from' => $message->getFrom(),
+                'to' => $t,
+                'source' => 'appwrite'
```

Comment:
Is the `source` param required? If not, let's remove it. If it is, let's set it to something like `Utopia PHP Messaging` to avoid coupling with Appwrite 👍

Reply:
is not required. I have removed it.